Vagrantfile

```diff
@@ -114,11 +115,11 @@ ansible_provision = proc do |ansible|
     # Use monitor_address_block instead of monitor_interface:
     ansible.extra_vars.delete(:monitor_interface)
     ansible.extra_vars = ansible.extra_vars.merge({
-      cluster_network: "#{SUBNET}.0/16",
+      cluster_network: "#{CLUSTER_SUBNET}.0/16",
```

Comment:
It should be /16 since the private LAN on Linode is anywhere in 192.168.0.0/16.  It would probably be a good idea to fix this so SUBNET is actually a subnet rather than a partial address...